tests/fixtures/app/eslint.config.mjs

```diff
@@ -81,8 +81,10 @@ export default [
     files: [
       '**/*.cjs',
       'config/**/*.js',
+      'tests/dummy/config/**/*.js',
```

Comment:
is there simple way to exclude this for actual apps?

Reply:
We'd need to copy the files to the addon blueprint, rather than continuing to use the app's files

Reply:
thanks for explanation @bertdeblock and @NullVoxPopuli !